chapters/annotations.tex

```diff
@@ -968,7 +905,7 @@ \subsubsection{Mouse input}\label{mouse-input}
 
 \begin{example}
 \begin{lstlisting}[language=modelica]
-interaction={ OnMouseDownSetBoolean(on, true), OnMouseUpSetBoolean(on, false)};
+interaction = {OnMouseDownSetBoolean(on, true), OnMouseUpSetBoolean(on, false)};
```

Comment:
This is now so long in pfd that it likely should be two lines.

Reply:
Fixed.  Also removing that semicolon.  Please resolve after verifying that the semicolon shouldn't be there.